app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/AbstractDeviceSupport.java

```diff
@@ -142,6 +142,11 @@ public Context getContext() {
         return context;
     }
 
+    @Override
+    public boolean useBleScannerForReconnect() {
+        return false;
```

Comment:
You mean similar to "Enable automatic BLE reconnect", globally for all devices? I was thinking on a per-device configuration, since the Casio devices require this anyway.

Reply:
The default implementation could check the global configuration and the casio implemention could override to forefully enable it.

I agree a pretty device-specific thing. I think until we have better device specific configurabilities, we could make it a global option.